userguide/content/en/docs/Best practices/site-guidance.md

```diff
@@ -33,7 +33,7 @@ configuring your local editor to use 4 spaces when the **Tab** key is pressed.
 
 ## Linking
 
-By default, regular relative URLs in links are left unchanged by Hugo (they're still relative links in your site's generated HTML), hence some hardcoded relative links like `[relative cross-link](../../peer-folder/sub-file.md)` might behave unexpectedly compared to how they work on your local file system. You may find it helpful to use some of Hugo's built-in [link shortcodes](https://gohugo.io/content-management/cross-references/#use-ref-and-relref) to avoid broken links in your generated site. For example a `{< ref "filename.md" >}` link in Hugo will actually
+By default, regular relative URLs in links are left unchanged by Hugo (they're still relative links in your site's generated HTML), hence some hardcoded relative links like `[relative cross-link](../../peer-folder/sub-file.md)` might behave unexpectedly compared to how they work on your local file system. You may find it helpful to use some of Hugo's built-in [link shortcodes](https://gohugo.io/content-management/cross-references/#use-ref-and-relref) to avoid broken links in your generated site. For example a `{{< ref "filename.md" >}}` link in Hugo will actually
```

Comment:
You need to escape the shortcode in the text as otherwise Hugo thinks it actually *is* a shortcode and gets confused - so instead of: {{< ref "filename.md >}}
do: {{</* ref "filename.md" */>}}

Reply:
Thanks for spotting this by the way!

Reply:
Oops, that makes sense! I've updated it in f1ea53e.